Gruntfile.js

```diff
@@ -418,7 +418,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-gh-pages')
 
   // Run development server using grunt serve
-  grunt.registerTask('serve', ['clean:server', 'copy:server', 'connect:livereload', 'watch']);
+  grunt.registerTask('serve', ['clean:server', 'copy:server', 'compass:dev', 'connect:livereload', 'watch']);
```

Comment:
Isn't this already included in the watch task check under css, of the watch task definition!

Reply:
Now that you have fixed the compass:dev I don't think you need to call this in the serve task explicitly as it's already being done in the watch task :)